source/features/insert-summary-button.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import onetime from 'onetime';
+import delegate from 'delegate';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
```

Comment:
The suggested icon also need to be added... when approved

Reply:
Maybe I'm overlooking this, but what suggested icon are you referring to? 🤔

Reply:
I think I had commented that on a commit so it got lost. I had suggested this icon: https://octicons.github.com/icon/fold-down/

Reply:
Added the icon and wanted your thoughts on something before I commit and push!

I noticed that in your changes that you moved the button over to the `Add a list` category from its initial position in the `insert` category.

Initial placement:

<img width="402" alt="insert category" src="https://user-images.githubusercontent.com/8808097/53675790-bf446100-3c5f-11e9-8a0d-5ebfcdecac19.png">

Your placement:

<img width="400" alt="add a list category" src="https://user-images.githubusercontent.com/8808097/53675806-d8e5a880-3c5f-11e9-8f5c-616cf5deae2a.png">

Are you against having it in the initial placement? I figured it would be a good spot since the items in that section all share the functionality of either inserting the action by itself, or highlighting your existing text, and having it formatted based on whichever action you chose.

It's not something I'm trying to argue over, as I'm sure you had your reasons, just wanted your thought process on the placement change 😄

Reply:
> the items in that section all share the functionality of either inserting the action by itself, or highlighting your existing text, and having it formatted based on whichever action you chose.

**All** the buttons behave that way.

I don't remember my reasoning at that time 😅 but since we're cleaning up the toolbar in #1809, maybe we can add it in a group with fewer buttons, like the right-most one, before `.js-saved-reply-container`

Reply:
> **All** the buttons behave that way.

Valid point!

How does this look?

<img width="133" alt="screen shot 2019-03-01 at 9 01 48 pm" src="https://user-images.githubusercontent.com/8808097/53676259-6e376b80-3c65-11e9-9b0c-0158238a8740.png">

Reply:
I think it's fine :)